packages/terra-form-select/src/Select.jsx

```diff
@@ -32,6 +33,12 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   dropdownAttrs: PropTypes.object,
+  /**
+   * Whether the select input should use the filter style display, forcing a value to always be selected.
+   * This also removes the placeholder and removes the ability for user to clear the value, returning the select to browser-native behavior.
+   * This is only applicable to the `default` variant.
+   */
+  isFilterStyle: PropTypes.bool,
```

Comment:
To be consistent with native select I think we should keep the prop name the same, but I'm having a hard time understanding how the prop name correlates to the functionality it toggles on and off. I know what the prop does, but I don't know what the prop name actually signifies.

Reply:
+1, yup we should provide a prop name that correlates with the functionality.

Reply:
+1, we should keep the prop name that correlates with the functionality